config/index.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   },
   "dev": {
     "name": "dev",
-    "dbhost": "localhost",
+    "dbhost": process.env.DB_PORT_6379_TCP_ADDR || 'localhost',
```

Comment:
this environment variable is set by docker when linking the redis container with the node container, and allows the node container to communicate with redis.

As in,

```
sudo docker run --name kaha_dev -v $(pwd):/kaha -p 3000:3000 --link redis:db kaha 
```

(db is the alias name and thus the prefix of the envirnonment var)